src/JabbR-Core/Services/InMemoryRepository.cs

```diff
@@ -110,6 +101,7 @@ public void Add(Attachment attachment)
 
         public void Add(ChatRoom room)
         {
+            RoomList.Add(room);
```

Comment:
Talked to heather about it and got some insight.. RoomList can be deleted. It is the same concept as _rooms. They created RoomList because they were not using Safe Collections before. Once SafeCollections was brought over to JabbrCore they stopped using RoomList and started using _rooms. So they only one needed is _rooms.